wolfcrypt/src/ecc.c

```diff
@@ -5547,7 +5594,11 @@ static int _ecc_make_key_ex(WC_RNG* rng, int keysize, ecc_key* key,
         DECLARE_CURVE_SPECS(ECC_CURVE_FIELD_COUNT);
 
         /* setup the key variables */
-        err = mp_init(&key->k);
+#ifndef ALT_ECC_SIZE
+        err = mp_init(key->k);
+#else
+        alt_fp_init(key->k);
```

Comment:
Should there be a return code check here in case init wasn't called and ->k isn't set to `->ka`?

Reply:
Change to set key->k to be key->ka before call.